cli/package.json

```diff
@@ -71,7 +71,6 @@
     "@types/chai": "4.2.15",
     "@types/chai-jquery": "1.1.40",
     "@types/jquery": "3.3.31",
-    "@types/lodash": "4.14.168",
```

Comment:
isn't @types/lodash a devDependency of cli?

https://github.com/cypress-io/cypress/blob/64b1aef36d3c801c76efc906b98a100a1079c0d7/cli/scripts/utils.js#L1-L8

Reply:
Was wondering why that was there, surprised nothing caught me.